app/views/ers_main.scala.html

```diff
@@ -34,6 +35,7 @@
     <meta name="format-detection" content="telephone=no"/>
     <meta http-equiv='Pragma' content='no-cache'/>
     <meta http-equiv='Cache-Control' content='no-cache, no-store, must-revalidate, private'/>
+    @hmrcTimeoutDialogHelper(signOutUrl = applicationConfig.signOut)
```

Comment:
Axe finds a few accessibility issues with the timeout dialog, not sure if you had a look?